java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -208,6 +209,16 @@ private static Object[] stringCommands(BaseTransaction<?> transaction) {
         String stringKey7 = "{StringKey}-7-" + UUID.randomUUID();
         String stringKey8 = "{StringKey}-8-" + UUID.randomUUID();
 
+        Map<String, Object> expectedLcsIdxObject =
+                Map.of("matches", new Object[] {new Long[][] {{1L, 3L}, {0L, 2L}}}, "len", 3L);
+
+        Map<String, Object> expectedLcsIdxWithMatchLenObject =
+                Map.of(
+                        "matches",
+                        new Object[] {new Object[] {new Long[] {1L, 3L}, new Long[] {0L, 2L}, 3L}},
```

Comment:
Here you have `new Object[] {new Object[] {new Long[] ...`, should be `Long[][][]` again